hackathon/spacecraft/src/hooks/usePlanets.ts

```diff
@@ -8,7 +8,7 @@ export const fetchPlanets = async () => {
 
 export const usePlanets = () => {
   return useQuery({
-    queryKey: ["planets"],
     queryFn: fetchPlanets,
+    queryKey: ["planets"],
```

Comment:
I think this is annoying 

because from the `tanstack-query` docs the order of the params is 

```
useQuery(queryKey, queryFn)
```

![CleanShot 2024-09-04 at 09 02 33](https://github.com/user-attachments/assets/3c653017-e4b4-40ae-ab4f-2c05a68d586d)

Can you try this setup?
https://perfectionist.dev/configs/recommended-natural